common/lib/xmodule/xmodule/course_module.py

```diff
@@ -834,6 +834,14 @@ class CourseFields(object):
         scope=Scope.settings,
     )
 
+    social_sharing_url = String(
+        display_name=_("Social Media Sharing URL"),
```

Comment:
@srpearce  - Tagging you here for the new studio setting being added by the edx.org support for social sharing icons.